iree/test/e2e/xla_ops/BUILD

```
@@ -28,8 +29,55 @@ package(
 
 iree_check_single_backend_test_suite(
     name = "check_cuda",
-    srcs = glob(
-        ["*.mlir"],
+    srcs = enforce_glob(
+        [
```

Might want to throw in some `# keep sorted` hints to buildifier here, so these lists stay sorted

```suggestion
        # keep sorted
        [
```